index.html

```diff
@@ -16,13 +16,13 @@
       </form>
       <div class="task-list">
         <div class="task-header">
-          <select name="" id="">
-            <option value="">title asc</option>
-            <option value="">title desc</option>
-            <option value="">due date asc</option>
-            <option value="">due date desc</option>
-            <option value="">creation date asc</option>
-            <option value="">creation date desc</option>
+          <select onchange="app.getSelectedValue()" name="" id="select_id">
```

Comment:
Avoid put instructions js in html. Is recomendable have the js logic separate as long as posible.